portality/static/js/application_form.js

```diff
@@ -380,6 +374,11 @@ doaj.af.TabbedApplicationForm = class extends doaj.af.BaseApplicationForm {
         }).fail(() => {
             // $("#validated-" + this.currentTab).val("False");
             this.tabValidationState[this.currentTab].state = "invalid";
+            let that = this;
+            let errFields = $(this.parsley.fields).filter(function(){return (this.validationResult !== true && this.domOptions.group === "block-" + that.currentTab)});
+            let errFirst = $(errFields.first()[0].element);
+            errFirst.trigger("focus");
+            errFirst.trigger("focus");
```

Comment:
I think it would be worth adding a bit of detail in a comment as to what this is fixing and why, since it's a bit weird.

Reply:
I did a bit more reading around this, and this might work with one hit: (from https://api.jquery.com/focus/):

`errFirst.triggerHandler("focus")`